vehicle-simulator/README.md

```diff
@@ -0,0 +1,110 @@
+
+```json
+{
+   "topic":"<YourDeviceId>:Vehicle/things/twin/commands/modify",
+   "headers":{
+      "content-type":"application/json",
+      "response-required":false
+   },
+   "path":"/features/OBD",
+   "value":{
+      "properties":{
+         "controlModuleVoltage": 9,
+         "b1601": true,
+         "b1602": false,
+         "b1603": true
+      }.
+   }
+}
+```
+
+Another data sent from the simulator but only after some user interaction, which will be explained later is a ditto feature called *trunk*. It contains information if the trunk is open or closed. A sample message for the trunk state looks like this.
+
+```json
+{
+   "topic":"<YourDeviceId>:Vehicle/things/twin/commands/modify",
+   "headers":{
+      "content-type":"application/json",
+      "response-required":false
+   },
+   "path":"/features/trunk",
+   "value":{
+      "properties":{
+         "isOpen":true
+      }
+   }
+}
+```
+
+# Commands
+
+The simulator supports the following commands.
+
+- Open Trunk
+- Reset DTC
+
+## Open Trunk
+
+When Open Trunk command is received, the simulator sends a *trunk* feature update, reporting *"isOpen" : true*, simulating opening of the trunk. After predefined, but configurable amount of time, the simulator sends *trunk* feature update, reporting *"isOpen" : false*, simulating closing of the trunk.
+
+To invoke the command one needs to send a message from the cloud instance to the corresponding connected device, where the simulator is running. The exact way to do this depends on the cloud itself, but the message payload has to be in the following ditto format.
+
+```json
+{
+  "topic": "<YourDeviceId>:Vehicle/things/twin/commands/openTrunk",
+  "headers": {
+  "content-type": "application/json",
+    "response-required": false
+  },
+  "path": "/"
+}
+```
+
+## Reset DTC
+
+When Reset DTC command is received, the simulator stops the random generation of the three DTCs (b1601, b1602, b1603). They are still sent periodically while updating the *OBD* feature, including the *controlModuleVoltage*, but with constant *false* value, simulating normal working of the vehicle and no trouble codes being raised. After the predefined timeout (the same as for the open trunk command) expires the simulator returns to the regular working mode and starts generating random boolean values for the three DTCs.
+
+To invoke the command one needs to send a message from the cloud instance to the corresponding connected device, where the simulator is running. The exact way to do this depends on the cloud itself, but the message payload has to be in the following ditto format.
+
+```json
+{
+  "topic": "<YourDeviceId>:Vehicle/things/twin/commands/resetDtc",
+  "headers": {
+  "content-type": "application/json",
+    "response-required": false
+  },
+  "path": "/"
```

Comment:
Is this message to the thing or to the DTC feature? The path should be aligned per the case, for reference check the [Ditto spec](https://eclipse.dev/ditto/protocol-specification-things-messages.html).